src/com/cdd/bao/editor/SearchSchema.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * BioAssay Ontology Annotator Tools
+ * 
+ * (c) 2014-2017 Collaborative Drug Discovery Inc.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License 2.0
+ * as published by the Free Software Foundation:
+ * 
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+package com.cdd.bao.editor;
+
+import java.util.*;
+
+import com.cdd.bao.editor.EditSchema.Branch;
+import com.cdd.bao.template.Schema;
+
+import javafx.scene.Node;
+import javafx.scene.control.TreeItem;
+import javafx.scene.control.TreeView;
+
```

Comment:
In between imports & top level class definition, add a short paragraph description of the class, to give the reader an idea of what it's for, and whether they should continue reading (see any other file in the project).